### DISCUSSION TITLE
How to store token/pat securely on Ubuntu 20.04 ?

### DISCUSSION THREAD

**Discussion by Ismoh** [nektos/act#1373] [Q&A] [answered]
I am new to storing sensitive data on Ubuntu.
I had a look, but I am still not sure how and where to store my GitHub PAT.

Should I use libsecret? (Is this is even possible?)
Should I use export PAT="askjdha213" ? (Is this really secure?) Dont think so.

How do you guys store your sensive token, when using act?

**Comment by KnisterPeter** [comment, marked as answer]
Since it is only required while act is running, setting it for that time only is enough.
No need to save a PAT for longer.

**Reply by Ismoh** [reply]
But I dont want to regenerate the token, everytime I want to have a quick look on local actions.
Any better idea?

**Reply by KnisterPeter** [reply]
You can encrypt it using gpg. Or use a desktop keyring or password store.

**Reply by Ismoh** [reply]
I am using pass with act -s now. Thanks a ton!
